setup/rt_resource_template_doc.json

```diff
@@ -177,12 +177,12 @@
       ],
       "http://www!w3!org/2000/01/rdf-schema#label": [
         {
-          "@value": "Label"
+          "@value": "Template Label"
         }
       ],
       "http://sinopia!io/vocabulary/hasRemark": [
         {
-          "@value": "The resource's label."
+          "@value": "The resource template's label."
```

Comment:
Should we remove "resource"? It seems that @michelleif is trying to move away from it.

Reply:
Updated with the latest commit.